sdk/keyvault/azkeys/autorest.md

```diff
@@ -0,0 +1,23 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml
+go: true
+version: "^3.0.0"
+input-file:
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/common.json
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/rbac.json
+- https://github.com/Azure/azure-rest-api-specs/blob/ecdce42924ed0f7e60a32c74bc0eb674ca6d4aae/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/securitydomain.json
+license-header: MICROSOFT_MIT_NO_VERSION
+clear-output-folder: false
```

Comment:
If internal/generated is purely generated code, suggest making this `true` to eliminate vestigial cruft